src/components/DragAndDropSubthoughts.tsx

```diff
@@ -104,7 +105,9 @@ const drop = (props: VirtualThoughtProps, monitor: DropTargetMonitor) => {
   const parentIdFrom = head(rootedParentOf(state, thoughtsFrom))
   const parentIdTo = head(rootedParentOf(state, pathTo))
   const sameContext = parentIdFrom === parentIdTo
-  const dropTop = attributeEquals(state, parentIdTo, '=drop', 'top')
+  const isExpanded = isThoughtExpanded(state, parentIdTo)
```

Comment:
Since we already have the `Path`, it would be better to make an `isPathExpanded` function. This avoids converting the `Path` to a `ThoughtId` and then back to a `Path`:

```ts
const isPathExpanded = (state: State, path: Path) =>
  !!state.expanded[hashPath(path)]

const isExpanded = isPathExpanded(rootedParentOf(state, pathTo))
```